monero-outreach-docs/translations/es/storyboard/randomX_es.md

```diff
@@ -0,0 +1,35 @@
+# RandomX
+*06/05/19*  
+_**Monero y Arweave a validar el algoritmo de Prueba-de-Trabajo.**_  
+ 
+¿Qué año es? Parece que solo fue ayer cuando decían que una moneda privada no podía reducir su tamaño de almacenamiento y que los ASICs ganarían al final. Pero aquí estamos, con los cascos espaciales puestos...reduciendo el tamaño del almacenamiento y hablando de RandomX. RandomX aún debe pasar por un proceso de prueba y autenticidad, pero es una parte emocionante del desarrollo de Monero. Esta guía da una descripción general de su naturaleza y plan, y, para que puedas aprender más. 
```

Comment:
Suggestion:

+ _Parece que fue **sólo ayer**_. Anyway, _solo_ must be changed to _sólo_.

We could keep the formatting of the original sentence and do something like this, what do you think?

+ _Pero aquí estamos... reduciendo el tamaño del almacenamiento y hablando de RandomX... con los cascos espaciales puestos._

Reply:
I first translated the sentence exactly as you suggested. While reading it, was hard to understand due to the "american" english association of space helmets as an american culture concept, even though I used space helmets in the sentence, I tried to give it a meaning too. I placed it in the first part of the sentence so the meaning had a closer understanding of space helmet culture. 

Not sure if you understand what I say. In latinamerica we are not used to use phrases that associate to space, but I will add this suggestion. Just wanted to give you my personal ambiguity.

Reply:
changed >:)